src/index.ts

```diff
@@ -38,4 +38,6 @@ mongoose
         console.log(`Failed to connect to MongoDB database: ${e}`);
     });
 
-initDiscord();
+if (process.env.RUN_DISCORD === 'true') {
```

Comment:
```suggestion
if (process.env.ENABLE_DISCORD_BOT === 'true' && process.env.DISCORD_TOKEN !== undefined) {
```

Slightly better error handling for missing tokens